python/lsst/ts/electrometer/commands_factory.py

```diff
@@ -214,28 +243,48 @@ def format_trac(
             The generated command string.
         """
         isFirst = True
-        if not (timestamp or temperature or channel or voltage):
-            command = ":trac:elem NONE"
+        if not (timestamp or temperature or voltage or set_mode):
+            if self.brand == "Keysight":
+                command = ":form:elem:sens NONE"
+            else:
+                command = ":trac:elem NONE"
         else:
-            command = ":trac:elem "
+            if self.brand == "Keysight":
+                command = ":form:elem:sens "
+            else:
+                command = ":trac:elem "
             if channel:
                 isFirst = False
                 command += "CHAN"
             if timestamp:
                 if not isFirst:
                     command += ", "
                 isFirst = False
-                command += "TST"
+                if self.brand == "Keysight":
+                    command += "TIME"
+                else:
+                    command += "TST"
             if temperature:
                 if not isFirst:
                     command += ", "
                 isFirst = False
-                command += "ETEM"
+                if self.brand == "Keysight":
+                    command += "TEMP"
+                else:
+                    command += "ETEM"
             if voltage:
                 if not isFirst:
                     command += ", "
                 isFirst = False
-                command += "VSO"
+                if self.brand == "Keysight":
+                    command += "SOUR"
+                else:
+                    command += "SOUR"

```

Comment:
This `if` `else` doesn't seem necessary, unless there is something missing here.

Reply:
Ooops, typo. I'll fix that.